blockchain/ethereum/gas price triggerer/2952c2ae-9f48-446a-b2d7-6d7e26b8b79e.json

```diff
@@ -0,0 +1,159 @@
+{
+  "name": "blockchain/ethereum/Gas price triggerer",
+  "description": "Sends a query to the given RPC node provider about the estimated gas price.\n(The gas price is determined by the last few blocks median gas price.)\n\nIt keeps polling the provider until the observed gas price is equal or under a specified limit.",
+  "interface": {
+    "inputs": {
+      "3f02e1e7-4750-42ed-ae51-044f6f6d7694": {
+        "type": "(\n  {\"providerUrl\": string} \n  or string\n)",
+        "name": "provider url",
+        "description": "Receives the connection string to a node provider.\nThe received data can be single string or can be a wider credentials record that has a providerUrl field."
+      },
+      "11a940f6-291f-4aaa-895e-57167ba88e2a": {
+        "type": "{\n\"price limit\": number,\noptional \"polling interval\": number,\noptional \"auto delay\": boolean\n}",
```

Comment:
Based on the testing every field has to be mandatory.

Reply:
agree, removed optional qualifiers